r_markdown/load_save_data.Rmd

```diff
@@ -211,6 +211,43 @@ Reference for test id, stove and fuel types.
                              "Liquefied Petroleum Gas")))
 ```
 
+* Define stove fuel combination category levels
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  samples <- dplyr::mutate(samples, stove_fuel = as.factor(paste(stove, ":", fuel))) %>%
+             dplyr::mutate(stove_fuel = factor(stove_fuel,
+                           levels = c("Three Stone Fire (Artisan) : Douglas Fir",
```

Comment:
Two variables in one variable! They haven't invented an appropriate emoji for this situation yet!